bootstrap2.go

```diff
@@ -0,0 +1,100 @@
+package coopdatadog
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/coopnorge/go-datadog-lib/v2/internal"
+	datadogLogger "github.com/coopnorge/go-logger/adapter/datadog"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
+	"gopkg.in/DataDog/dd-trace-go.v1/profiler"
+)
+
+// Start the Datadog integration, use the returned context.CancelFunc to stop
+// the Datadog integration.
+func Start(ctx context.Context, options ...Option) (context.Context, context.CancelFunc, error) {
+	if internal.IsDatadogDisabled() {
+		return ctx, context.CancelFunc(func() {}), nil
+	}
+
+	err := internal.VerifyEnvVarsSet(
+		internal.DatadogAPMEndpoint,
```

Comment:
In case of using TCP for APM (might also be for DSD) we also need require `DD_AGENT_HOST
ref: https://docs.datadoghq.com/containers/kubernetes/apm/?tab=tcp#configure-your-application-pods-to-submit-traces-to-datadog-agent

Reply:
Is this something we want to official support in the context of the platform? This would require helm charts or other official ways of deploying containers to provide that environmental variable.

Reply:
Well this ad least add support for running outside of the kubernetes since only there the socket is supported. We used to be able to run the library in TCP mode was well.

Reply:
( i used TCP mode  before, might be a bit confusing with the library using HTTP)

Reply:
The function only verify that the required environmental variables are defined. Optional ones are still supported by the underlying library.

Let's check if both of these can be required at the same time and we get the correct behavior. If not the have to be exclusive.

Reply:
https://github.com/coopnorge/go-datadog-lib/commit/59fe6e7974c70d7e89309b1025eaff8126fbf5f7 deprecated `ConnectionTypeHTTP` and `ConnectionTypeSocket`. https://github.com/coopnorge/go-datadog-lib/commit/6c339fac254a9e5c7531ac90d6d56311dfa5bf98 leaves it to `ddtrace` to be responsible for loading the URL from `DD_TRACE_AGENT_URL`.

I am a bit confused here, should we require both `DD_TRACE_AGENT_URL` and `DD_AGENT_HOST` to be set or 1 of them?

Reply:
Both should be set... It is very confusing..

Reply:
OK, I'll add `DD_AGENT_HOST` as a required environmental variable as well

Reply:
added